src/pages/groupComparison/GroupComparisonMutationMapper.tsx

```diff
@@ -7,32 +7,107 @@ import {
+import { extractColumnNames } from 'shared/components/mutationMapper/MutationMapperUtils';
+import autobind from 'autobind-decorator';
+import { Sample } from 'cbioportal-ts-api-client';
+import { FisherExactTwoSidedTestLabel } from './FisherExactTwoSidedTestLabel';
+import ComplexKeyMap from 'shared/lib/complexKeyDataStructures/ComplexKeyMap';
+import { CheckedSelect, Option } from 'cbioportal-frontend-commons';
+import { SHOW_ALL_PAGE_SIZE as PAGINATION_SHOW_ALL } from '../../shared/components/paginationControls/PaginationControls';
 
 interface IGroupComparisonMutationMapperProps extends IMutationMapperProps {
     onInit?: (mutationMapper: GroupComparisonMutationMapper) => void;
     axisMode?: AxisScale;
     onScaleToggle?: (selectedScale: AxisScale) => void;
     groups: ComparisonGroup[];
     annotationFilterSettings: IAnnotationFilterSettings;
+    groupToProfiledPatients: {
+        [groupUid: string]: string[];
+    };
+    sampleSet: ComplexKeyMap<Sample>;
+    profiledPatientCounts: number[];
 }
 
 @observer
 export default class GroupComparisonMutationMapper extends MutationMapper<
     IGroupComparisonMutationMapperProps
 > {
+    @observable.ref _selectedGroupsForEnrichedInFilter: string[];
+    @observable significanceFilterEnabled: boolean = false;
+
     constructor(props: IGroupComparisonMutationMapperProps) {
         super(props);
+        makeObservable(this);
+
+        this._selectedGroupsForEnrichedInFilter = this.props.groups.map(
+            group => group.nameWithOrdinal
+        );
+    }
+
+    @computed get paginationStatusText(): string {
+        let firstVisibleItemDisp;
+        let lastVisibleItemDisp;
+        let dataStore = this.props.store.dataStore as MutationMapperDataStore;
+
+        if (dataStore.tableData.length === 0) {
+            firstVisibleItemDisp = 0;
+            lastVisibleItemDisp = 0;
+        } else {
+            firstVisibleItemDisp =
+                dataStore.itemsPerPage === PAGINATION_SHOW_ALL
+                    ? 1
+                    : dataStore.page * dataStore.itemsPerPage + 1;
+
+            lastVisibleItemDisp =
+                dataStore.itemsPerPage === PAGINATION_SHOW_ALL
+                    ? dataStore.visibleData.length
+                    : firstVisibleItemDisp + dataStore.visibleData.length - 1;
+        }
+
+        // if there are multiple mutations represented, use Mutations else Mutation in label
+        const mutationsLabel =
+            _.flatten(dataStore.tableData).length === 1
+                ? 'Mutation'
+                : 'Mutations';
+
+        // text originally shows Mutation(s) based on rows, since the rows represent protein changes now, replace Mutation(s)
+        // with Protein Change(s) with {mutationsLabel}
+        let itemsLabel =
+            dataStore.tableData.length === 1
+                ? `Protein Change with ${mutationsLabel}`
+                : `Protein Changes with ${mutationsLabel}`;
```

Comment:
Do we need this much granularity? How about just 
```suggestion
        const itemsLabel = `Protein Change(s) with ${mutationsLabel}`;
```